.travis.yml

```diff
@@ -10,12 +10,15 @@ before_install:
 install:
   - sudo sysctl -w vm.max_map_count=262144
 
-before_script: greenkeeper-lockfile-update
+before_script:
```

Comment:
Why the change here (and the other 2 instances)?

I've been moving to having single lines as they are in order to reduce the length of the file.